IntentsExtension/Support Files/IntentsExtensionDebug.entitlements

```diff
@@ -8,6 +8,12 @@
 	<array>
 		<string>$(TeamIdentifierPrefix)com.automattic.SimplenoteMac.Development</string>
 	</array>
+	<key>com.apple.security.files.user-selected.read-write</key>
+	<true/>
+	<key>com.apple.security.network.client</key>
+	<true/>
+	<key>com.apple.security.network.server</key>
+	<true/>
```

Comment:
Same note as above, we should probably remove the `.server` entitlement (since we don't require inbound connections)